src/main/java/org/sonar/plugins/stash/issue/MarkdownPrinter.java

```diff
@@ -120,12 +130,50 @@ private String formatTableList(String caption, List<String> items) {
     sb.append("|");
     sb.append(NEW_LINE);
     for (String item : items) {
-      sb.append("| ").append(item).append(" |").append(NEW_LINE);
+        sb.append("| ").append(item).append(" |").append(NEW_LINE);
+        if (this.includeFilesInOverview > 0 && issueToFilesMap.containsKey(item)) {
+            String files = String.join(", ", issueToFilesMap.get(item));
+            sb.append("| :arrow_right_hook: *Files: ").append(files).append("* |").append(NEW_LINE);
+        }
     }
 
     return sb.toString();
   }
 
+  private ListMultimap<String, String> getMappedIssuesToFiles(Collection<PostJobIssue> allIssues) {
+    ListMultimap<String, String> issueToFilesMap = ArrayListMultimap.create();
+    if (this.includeFilesInOverview <= 0) {
+      return issueToFilesMap;
+    }
+
+    for (PostJobIssue issue : allIssues) {
+      String title = this.printIssueMarkdown(issue);
+
+      Integer issueLine = issue.line();
+      if (issueLine == null) {
+        issueLine = 0;
+      }
+
+      issueToFilesMap.get(title).add(this.issuePathResolver.getIssuePath(issue) + ":" + issueLine);

```

Comment:
Would it be possible to key the dictionary by something else?
I think `issue.ruleKey()` should work.

Reply:
The problem is that where we use this information we only have the title of the issues:

```java
private String formatTableList(List<String> items, ListMultimap<String, String> issueToFilesMap)
```

It will require refactoring of 

```java
List<String> uniqueSortedInformation = allIssues.stream()
              .filter(distinctByKey(PostJobIssue::ruleKey))
              .sorted(Comparator.comparing(PostJobIssue::severity).reversed())
              .map(this::printIssueMarkdown)
              .collect(Collectors.toList());
```

Reply:
I strongly prefer a refactoring, but can also perform it afterwards myself. Your choice.
It should be possible to to `collect()` into `Multimaps.toMultimap()`. And do the formatting and sorting afterwards.

Reply:
As not a JAVA developer that seems like pretty complex magic and would not prefer to do this refactoring on this PR. But i can check it out later and create different PR for that.

Reply:
Ok, deal.

Reply:
@pbojan Are you currently doing or planning to do this refactoring? If not, I will do it.

Reply:
Not at the moment so please do. Any idea in what release this feature will be available?

Reply:
Okay.
This would be in 1.5.0, which has currently no planned release.
(We are in rc-phase of 1.4.0)